lib/alglie.gi

```diff
@@ -3776,6 +3776,45 @@ end );
 ##  For the algorithm we refer to C. Reutenauer, Free Lie Algebras, Clarendon
 ##  Press, Oxford, 1993.
 ##
+
+DeclareGlobalFunction( "IsLyndonT" );
+
+InstallGlobalFunction( IsLyndonT,
```

Comment:
I would prefer not to introduced new uses of `DeclareGlobalFunction` (which introduces a "placeholder" object) esp. for internal helpers. This should work, too:
```suggestion
DeclareGlobalName( "IsLyndonT" );

BindGlobal( "IsLyndonT",
```